### ISSUE TITLE
Disabling/Enabling Modules Doesn't Register/Unregister Blocks/Items

### ISSUE THREAD

**Issue by sylv256** [ProtonMC/Proton#36]
:concern:
also, mixin to datafixerupper so it doesn't delete blocks/items from disabled modules.

**Comment by YTG1234** [comment]
Are you reporting a bug or suggesting a change? The title and comment make it very unclear.

**Comment by sylv256** [comment]
Bug

**Comment by YTG1234** [comment]
> Disabling/Enabling Modules Doesn't Register/Unregister Blocks/Items

You can't register/unregister thingies after the mod initializes. That's how Minecraft's registry works.

**Comment by dzwdz** [comment]
couldn't we always register the blocks, and just remove their crafting recipes/their itemgroups when they get disabled?

**Comment by namebee** [comment]
I don't think this would work, And if it does it would be really sketchy to do so.

**Comment by dzwdz** [comment]
so, what if someone with all modules disabled joins a server with all modules enabled? if we don't register the blocks, they won't be able to join
also, it will make enabling/disabling blocks at runtime possible
